src/store/reducers/dform/initial/styles.json

```diff
@@ -2978,7 +2978,171 @@
         },
         {
           "class": "block",
-          "label": "Gynecology",
+          "label": "SEROLOGICAL TESTS",
+          "children":[
+            {
+              "field": "serological.test.HIV",
```

Comment:
I think this can be `serological_test.HIV` or `serological.HIV` while, yes, this works.

`field` specifies which field in a document (item) of CouchDB is used to store the data.
CouchDB stores every records in JSON-like format.
(You can see it via the CouchDB's dashboard as below.)

So, for example, when you set `field` as `serological.test.HIV`, that data will be stored here:
```
{
  "serological": {
    "test": {
      "HIV": "THE DATA"
    }
  }
}
```
but the level of "test" is not necessary.

------
Appendix: How to check the stored data via CouchDB's dashboard

<img width="993" alt="2018-07-18 20 52 34" src="https://user-images.githubusercontent.com/3135397/42879821-f31991ac-8acc-11e8-9cc6-e70db01752f8.png">
<img width="979" alt="2018-07-18 20 52 53" src="https://user-images.githubusercontent.com/3135397/42879823-f3433656-8acc-11e8-9043-7db37370afe8.png">
<img width="986" alt="2018-07-18 20 53 00" src="https://user-images.githubusercontent.com/3135397/42879825-f36990c6-8acc-11e8-9ac5-c438a853b861.png">